### PR TITLE
feat: added eth_BlobBaseFee

### PR DESCRIPTION
**Pull request by varun-doshi**
Ref #200 

Added `eth_blobBaseFee`

### CODE REVIEW

File: core/src/client/rpc.rs [hunk 1]
```diff
@@ -84,6 +84,8 @@ trait EthRpc<TX: TransactionResponse + RpcObject, TXR: RpcObject, R: ReceiptResp
     async fn gas_price(&self) -> Result<U256, ErrorObjectOwned>;
     #[method(name = "maxPriorityFeePerGas")]
     async fn max_priority_fee_per_gas(&self) -> Result<U256, ErrorObjectOwned>;
+    #[method(name = "getBlobBaseFee")]
```
Comment (eshaan7) [inline core/src/client/rpc.rs:87]: As per the [spec](https://ethereum.github.io/execution-apis/api-documentation/), it is `eth_blobBaseFee` so the `get` is not required.

**Review by ncitron — CHANGES_REQUESTED** [review]
Also there are some clippy and fmt errors introduced here related to unused imports.

**Comment by eshaan7** [comment]
nit: need to update method name in PR title as well.

File: core/src/client/rpc.rs (hunk 1 above)
Comment (ncitron) [inline core/src/client/rpc.rs:87]:
```suggestion
    #[method(name = "blobBaseFee")]
```

**Pull request merged**